### ISSUE TITLE
NVDA does not read labels of fields in dragon's add new command dialogue

### ISSUE THREAD

**Issue by pranavlal** [dictationbridge/dictationbridge-nvda#8]
NVDA does not read the correct labels of the fields when you tab through the "add new command" dialogue. To duplicate this, take the following steps.
#Launch Dragon.
#Navigate to the tools menu and select the option to add a new command.
#Tab through the dialogue box and note the labels spoken. Labels for the fields for the type of command to create and the state of the command are not spoken correctly.

**Comment by derekriemer** [comment]
this probably won't make the 1.0 milestone.
